### PR TITLE
Make new eventing as a default component

### PR DESCRIPTION
**Pull request by radufa**
**Description**
The new Eventing component replaces the Knative-Eventing components. 
It supports two messaging backend systems: BEB and NATS.

Changes proposed in this pull request:

- Add eventing component
- Remove Knative-Eventing

**Related issue(s)**

### CODE REVIEW

File: resources/eventing/values.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@ postUpgradeJob:
     name: event-sources-upgrade-job
     tag: PR-10733
```
Comment (marcobebway) [inline resources/eventing/values.yaml:37]: Please update the `postUpgradeJob.image.tag` with the final image (if available).

**Review by marcobebway — COMMENTED** [review]
Please delete the following files form Kyma repo:

- https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-config-azure-eventhubs.yaml.tpl
- https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-cr-azure-eventhubs.yaml.tpl

**Review by sayanh — DISMISSED** [review]

File: resources/eventing/values.yaml (hunk 1 above)
Comment (sayanh) [reply resources/eventing/values.yaml:37]: Fixed

**Comment by sayanh** [comment]
@marcobebway Sure, let's solve those in this [issue](https://github.com/kyma-project/kyma/issues/10891).

**Review by marcobebway — DISMISSED** [review]

**Review by dariadomagala — DISMISSED** [review]

**Review by franpog859 — DISMISSED** [review]

**Review by tgorgol — DISMISSED** [review]

File: resources/cluster-essentials/files/subscriptions.eventing.kyma-project.io.crd.yaml [hunk 2]
```diff
@@ -15,6 +15,10 @@ spec:
   scope: Namespaced
   versions:
   - name: v1alpha1
+    additionalPrinterColumns:
+      - jsonPath: .status.ready
+        name: Ready
+        type: string
```
Comment (marcobebway) [inline resources/cluster-essentials/files/subscriptions.eventing.kyma-project.io.crd.yaml:21]: Thanks 👍

**Review by marcobebway — APPROVED** [review]

**Review by franpog859 — DISMISSED** [review]

**Review by franpog859 — APPROVED** [review]

**Review by tgorgol — APPROVED** [review]

**Review by marcobebway — APPROVED** [review]

**Review by dariadomagala — APPROVED** [review]

**Review by k15r — APPROVED** [review]

**Review by pbochynski — APPROVED** [review]

**Review by strekm — APPROVED** [review]
/meow

**Pull request merged**